### ISSUE TITLE
Feture request ;)

### ISSUE THREAD

**Issue by ib54003** [northox/roundcube-yubikey-plugin#8]
Hey maybe it would be cool to support FIDO ("https://fidoalliance.org/specifications/overview/") or HOTP/TOTP. Also a recovery funciton would be usefull in case u lost your yubikey ;).

**Comment by northox** [comment]
Hello ib54003,

I'd be interested in integrating FIDO U2F as it's a new standard that's getting a lot of push but currently don't have any cycles for this. Maybe in the future or if I get more request of it. For the record, here's a few reference:
- client lib: https://github.com/Yubico/u2fval-client-php
- other refs: https://security.stackexchange.com/questions/71316/how-secure-are-the-fido-u2f-tokens

I know Yubikey do support HOTP/TOTP but could you explain the value of doing so? So far, you're the only one who as been asking for this I'm not quite sure there's a big market for it.

Either way I'm open to Pull Request though. ;)

**Comment by K0HAX** [comment]
I'll add a +1 to requesting support for FIDO U2F.

**Comment by ib54003** [comment]
any news?

**Comment by northox** [comment]
Noting tangible no. So far, it seems like it would be better for me to fork this project into a separate one: either you use Yubico auth, or you use U2F. 

I might have some time during holiday vacation but no guarantee - I have other projects running https://mantor.org and https://danny.mantor.org/qubes-rumprun/.

**Comment by corbolais** [comment]
Hi,
I'm looking for a roundcube plugin featuring support for U2F. Be it via yubikey or solokeys [doc](https://docs.solokeys.io/) ([how to get](https://shop.solokeys.com/)).

The topic is already four years old.

Anything new to the repo in this regard?

cheers

**Comment by northox** [comment]
No. No time/intention to put invest on this but I welcome pull requests.
